src/main/java/com/zaxxer/sansorm/internal/Introspected.java

```diff
@@ -482,42 +482,31 @@ private static String readClob(Clob clob) throws IOException, SQLException
       }
    }
 
-   private void processColumnAnnotation(FieldColumnInfo fcInfo)
+   private void processFieldAnnotations(FieldColumnInfo fcInfo)
    {
       Field field = fcInfo.field;
 
       Column columnAnnotation = field.getAnnotation(Column.class);
       if (columnAnnotation != null) {
```

Comment:
Body extracted as method to make processFieldAnnotations() more overseeable and to reduce complexity.